app/views/recipes/_recipecard.html.erb

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
I think it would make more sense and follow established rails paradigms/best practices if we re-name this file to `_form.html.erb` or `_recipe_form.html.erb`. When I see "recipecard" I think of the card that we use to display the recipe.